http-server-handler/thumbnailator/execute-requests.go

```diff
@@ -0,0 +1,113 @@
+package main
+
+import(
+	"os"
+	"os/exec"
+	"log"
+	"strings"
+	"time"
+	"net/http"
+	"strconv"
+	"fmt"
+	"io"
+	"io/ioutil"
+)
+
+func main() {
+	protocol := "http://"
+	serverAddress := os.Args[1]
+	endpoint := os.Args[2]
+	var builder strings.Builder
+	builder.WriteString(protocol)
+	builder.WriteString(serverAddress)
+	builder.WriteString(endpoint)
+	functionUrl := builder.String()
+
+	nRequests, err := strconv.ParseInt(os.Args[3], 10, 64)
+	if err != nil {
+		log.Fatal(err)
+		os.Exit(-1)
+	}
+	executionId := os.Args[4]
```

Comment:
Dois comentários:

1. Mover essa linha para cima, é melhor que toda a validação dos parâmetros aconteça no início do programa (se o usuário não tiver passado os.Args[4] essa linha vai matar o programa se mensagem bacana).
1. Pelo mesmo motivo que falei lá em cima com relação a URL, essa variável deve se chamar `executionID`